proven-member/disclosure-lib/src/main/java/gov/pnnl/proven/cluster/lib/disclosure/message/DisclosureMessage.java

```diff
@@ -79,26 +84,71 @@
 	 * True, if the disclosed content is {@code MessageContent#Measurement}.
 	 */
 	boolean hasMeasurements;
+	
+	/**
+	 * mimeType can be application/json or application/jsonld.
+	 */
+	String mimeType;
 
 	public DisclosureMessage() {
 	}
 
-	public DisclosureMessage(JsonObject message) {
+	public DisclosureMessage(JsonObject message) throws Exception {
 		this(message, null);
 	}
 
-	public DisclosureMessage(JsonObject message, JsonObject schema) {
+	public DisclosureMessage(JsonObject message, JsonObject schema) throws Exception {
 		super(message, schema);
 
 		// TODO these are assumed default characteristics of the disclosed
 		// message for an initial implementation (i.e. explicit disclosure of
 		// measurement data). Need to add internal methods to examine the JSON
 		// to set the member properties based on the actual content of the
 		// message.
-		this.disclosedContent = MessageContent.Explicit;
+		
+		MessageModel mm = MessageModel.getInstance();
+		try {
+			String jsonApi = mm.getModelFile("proven-schema.json");
```

Comment:
Would you please create a new method in message model to retrieve the api schema file.  For example, **mm.getApiSchema()** .  This way, a string constant for the api schema file can be defined in the MessageModel and referred to in the new method to retrieve the file.